### ISSUE TITLE
Warning: Missing Terms of Service and/or Privacy policy URL, see `set_tos_url` and `set_privacy_policy_url` methods

### ISSUE THREAD

**Issue by kalehdoo** [JohnCoene/firebase#4]
Hello,

I am not able to find any documentation on how to add the URLs for policy and terms.

I have deployed my app on shinyapps.io.

Warning: Missing Terms of Service and/or Privacy policy URL, see `set_tos_url` and `set_privacy_policy_url` methods

The document only mentions that:
link to the document: https://firebase.john-coene.com/articles/ui.html
Notice the warning in the console; this should be fixed when deploying to prod.
 
Thanks,
Manohar

**Comment by Pit-Storm** [comment]
Check the Reference. Espacially this for [Terms of Service](https://firebase.john-coene.com/reference/firebaseui#method-set_tos_url) and this for [Privacy Policy](https://firebase.john-coene.com/reference/firebaseui#method-set_privacy_policy_url).

**Comment by pedroorden** [comment]
> You can use the `set_` methods to specify those https://firebase.john-coene.com/reference/firebaseui#method-set_tos_url

Hi! its not working

**Comment by JohnCoene** [comment]
Use the 

`set_tos_url` and `set_privacy_policy_url` methods.

What is not working?

**Comment by JohnCoene** [comment]
Did you find the methods?

**Comment by pedroorden** [comment]
no, there is something I am missing. I can't find a reference on how to deal with set_tos_url set_privacy_policy_url

**Comment by JohnCoene** [comment]
Can you share your code on what you tried?

```r
f <- FirebaseUI$new$set_tos_url("myurl")
```
